docs/source/checks/nlp/model_evaluation/plot_metadata_segments_performance.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
Can you open issues for all of the warnings so we fix the checks such that they'll work nicely without warnings for current and previous scikit-learn versions?
Seems that there are a few different ones.